_metadata.py

```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.25.0"
+__extension_version__ = "0.27.0"
```

Comment:
Looks like this was previously updated to 0.26 without a release, so we should keep it.

Reply:
done in @cqc-melf
[update version](https://github.com/CQCL/pytket-aqt/pull/27/commits/5105b1fd664cbae638534ccb6c5571bdb5e3bfd1)
[5105b1f](https://github.com/CQCL/pytket-aqt/pull/27/commits/5105b1fd664cbae638534ccb6c5571bdb5e3bfd1)